float64 x
float64 y
float64 z
float64 yaw
float64 mass
bool isChecked
uint32 buttonID
bool shouldCheckForAgentID
uint32[] agentIDs